ios/Panthalassa.m

```diff
@@ -545,11 +581,23 @@ -(void)stopObserving {
   hasListeners = NO;
 }
 
+// This method should be deleted due is not the active protocol listener now
```

Comment:
Is there a reason why we should keep that? If no, let's remove it right now.

Reply:
Done now